src/python/nimbusml/pipeline.py

```diff
@@ -2270,19 +2270,9 @@ def transform(
                 "Model is not fitted. Train or load a model before test("
                 ").")
 
-        if y is not None:
-            if len(self.steps) > 0:
-                last_node = self.last_node
-                if last_node.type == 'transform':
-                    raise ValueError(
-                        "Pipeline needs a trainer as last step for test()")
-
         X, y_temp, columns_renamed, feature_columns, label_column, \
             schema, weights, weight_column = self._preprocess_X_y(X, y)
 
-        if not isinstance(y, (str, tuple)):
```

Comment:
>tuple [](start = 35, length = 5)

when y could be a tuple? How y_temp is used now?

Reply:
y_temp is not used at all in this method.

---
In reply to: [330999871](https://github.com/microsoft/NimbusML/pull/294#discussion_r330999871) [](ancestors = 330999871)